cmake/Modules/ConfigureSYCLBLAS.cmake

```diff
@@ -44,6 +44,10 @@ if("double" IN_LIST BLAS_DATA_TYPES)
   add_definitions(-DBLAS_DATA_TYPE_DOUBLE)
 endif()
 
+if("cl::sycl::half" IN_LIST BLAS_DATA_TYPES)
```

Comment:
Are you sure this is a good name? Shouldn't this be just `half`?

Reply:
This is what I initially wanted to do: https://github.com/codeplaysoftware/sycl-blas/pull/245#discussion_r454552435. It's imporntant to have both a user facing name and the C++ type it maps to. There are multiple options to do it:
1. Have a conversion function
2. Convert user provided list into an internal one
3. Keep both lists

Maybe there's another option that I'm not seeing. I personally prefer 1. 2 would be pretty easy to do, but this line (and others) would stay the same. I explored 3 a little bit, but the connected between the lists seems very fragile: there are a few places in CMake where it's OK to use the user provided name, but also places where you must use the C++ type.

Reply:
having :: and namespace in the name does not seem convenient. Can you make it float16?

Reply:
Updated with a conversion function, the user would just use `half`: https://github.com/codeplaysoftware/sycl-blas/pull/251/commits/c49578f8c5a9a2de31374da18e08d0d16b6e34b9